routers/api/v1/org/repo.go

```diff
@@ -0,0 +1,36 @@
+package org
+
+import (
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	api "code.gitea.io/sdk/gitea"
+)
+
+// ListRepos list all of an organization's repositories.
+func ListRepos(ctx *context.APIContext) {
+	requester := ctx.User
+	org := ctx.Org.Organization
+
+	// Find all repos a user has access to within an org.
+	reposEnv, err := org.AccessibleReposEnv(requester.ID)
```

Comment:
Maybe you can add an integration test on integrations sub directory. There are some examples there.